package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "devDependencies": {
+        "markdownlint-cli": "^0.26.0"
+    },
+    "license": "UNLICENSED",
+    "private": true,
+    "scripts": {
+        "coding-standards-check/markdownlint": "yarn markdownlint --ignore LICENSE.md *.md docs/",
+        "coding-standards-check": "yarn coding-standards-check/markdownlint",
+        "coding-standards-apply/markdownlint": "yarn markdownlint --fix --ignore LICENSE.md *.md docs/",
+        "coding-standards-apply": "yarn coding-standards-apply/markdownlint"
+    },
+    "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.15.3",
+        "yarn": "^1.22.10"
+    }
+}
```

Comment:
Your editor should be configured to always add a newline at the end of a text file. And having it remove trailing whitespace from lines when saving a file is also a good thing.